### PR TITLE
API calls for general weather information

### PR DESCRIPTION
**Pull request by valkim55**
hey guys, pushing my feature branch with the code for weather api calls

### CODE REVIEW

**Review by kabirfaisal1 — APPROVED** [review]
looks good to me

File: assets/js/weatherSript.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        })
+        // show an alert in case there's network error
+        .catch(function(error) {
+            alert('unable to connect to server. try again later');
+        })
+        // empty the input field
+        locationInput.value = '';
+
+    // show an alert if user submits empty field    
+    } else {
```
Comment (mastalukeremix) [inline assets/js/weatherSript.js:40]: messy notation, but should work on paper

File: assets/js/weatherSript.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        var firstURL = 'https://api.openweathermap.org/data/2.5/weather?q='+locationName+'&appid='+apiKey+'&units=imperial';
+        fetch(firstURL).then(function(response) {
+            // check if entered location name actually exists
+            if(response.ok) {
+                response.json().then(function(data) {
+                    // call getGeoLocation to get longitude and latitude for the precise weather at a requested location
+                    console.log(data);
+                    getGeoLocation(locationName, data);
+                });
+            } else {
```
Comment (mastalukeremix) [inline assets/js/weatherSript.js:28]: might want to fix the else readability here

File: assets/js/weatherSript.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        // empty the input field
+        locationInput.value = '';
+
+    // show an alert if user submits empty field    
+    } else {
+        alert('Please enter a location name');
+    }
+    
+};
+
```
Comment (mastalukeremix) [inline assets/js/weatherSript.js:45]: ^ might want to fix some of the code organization on this if else statement here to improve readability.

**Review by mastalukeremix — COMMENTED** [review]
Overall looks good just some minor spacing issues imo.

**Pull request merged**